app/screens/draft_scheduled_post_options/delete_draft.tsx

```diff
@@ -62,6 +68,16 @@ const DeleteDraft: React.FC<Props> = ({
                 channelId,
                 rootId,
             });
+            return;
+        }
+        if (websocketState !== 'connected') {
+            showSnackBar({
+                barType: SNACK_BAR_TYPE.CONNECTION_ERROR,
+                customMessage: intl.formatMessage({id: 'network_connection.not_connected', defaultMessage: 'No internet connection'}),
```

Comment:
Really bad error message, this should definitely be changed, device may have internet but the connection to the websocket fails for whatever reason. Also, why rely on the WS connection state here?

Reply:
Removed the websocketState check, as we now depend on the server error message.